src/client/app/components/snippet.info.ts

```diff
@@ -16,6 +17,11 @@ import { Strings } from '../helpers';
                     <label class="ms-Label">${Strings.descriptionLabel}</label>
                     <textarea class="ms-TextField-field ms-font-m" [(ngModel)]="snippet.description" placeholder="Description of the snippet"></textarea>
                 </div>
+
+                <div *ngIf="!!url" class="ms-TextField">
```

Comment:
What if we instead had it as a link:

     Gist URL:  <a href="..." target="_blank">Open in browser</a>